src/app/wizard/views/Mesh.vue

```diff
@@ -499,13 +499,15 @@
                 <template #kubernetes>
                   <CodeBlock
                     id="code-block-kubernetes-command"
+                    data-test-codeblock="kubernetes"
```

Comment:
**Suggestion**: Can we stick to the commonly present `data-testid` style format here? I know it doesn’t matter, but I rather have _one_ attribute name with _different_ values_ than both name and value being different.

Reply:
Using a single one means they can easily get overridden:

![Screenshot 2022-11-16 at 14 17 37](https://user-images.githubusercontent.com/554604/202204511-beae5784-8f11-490e-a767-f6ecca657206.png)

Not sure if overwriting that existing `data-testid` does anything right now, but might be worth chatting through how to avoid this, say deciding on a pattern using `data-test-component` for top level component node? Do you know if Kongponents uses `data-test-*` attributes?

Let me know if you still want to tweak anything here (or change to `data-testid` anyway), otherwise I'll merge a little later.

Reply:
Why do you want to avoid overwriting them? In my experience, you rarely want to rely on attributes you don’t control. It’s also pretty rare that you even need to find and use components from external dependencies.

I don’t think any more sophistication is needed here. `data-testid="kubernetes-code-block"` does the job just as well. Happy to reconsider this is we do have such conflicts.

(Kongponents uses `data-testid="*"` attributes exclusively)

Reply:
> Why do you want to avoid overwriting them?

I suppose because I've no idea if something was relying on that elsewhere (in this instance I guess it's not as no tests broke).

> (Kongponents uses data-testid="*" attributes exclusively)

Ok good to know

Will switch up to use `data-testid`, gimme 2 tics